packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cdk-hotswap-deployment-for-ecs-service-detects-failed-deployment-and-errors.integtest.ts

```diff
@@ -10,7 +10,7 @@ integTest(
 
     // WHEN
     const deployOutput = await fixture.cdkDeploy('ecs-hotswap', {
-      options: ['--hotswap'],
+      options: ['--hotswap', '--hotswap-operation-timeout-seconds', '10'],
```

Comment:
🤢

`--hotswap-timeout=10s` ?

I know you probably want `seconds` in there to be quite clear... but this is an extremely long flag name to type.

Reply:
+1

Reply:
Changed to `--hotswap-timeout=10s` in the CLI layer only, APIs still accept a concrete number of seconds.

Reply:
Nice! That's exactly what we have the separate layers for.